paginate/src/main/java/com/paginate/abslistview/WrapperAdapter.java

```diff
@@ -67,6 +67,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
 
     void displayLoadingRow(boolean displayLoadingRow) {
         this.displayLoadingRow = displayLoadingRow;
+        wrappedAdapter.notifyDataSetChanged();
```

Comment:
I was looking around and found out that this is not the intention. In what case would you need to call notifyDataSetChanged upon wrapped adapter directly?